include/session/blinding.h

```diff
@@ -113,6 +113,22 @@ LIBSESSION_EXPORT bool session_blind25_sign(
         size_t msg_len,
         unsigned char* blinded_sig_out /* 64 byte output buffer */);
 
+/// Computes a verifiable version-blinded signature that validates with the version-blinded pubkey
+/// that would be returned from blind_version_key_pair.
+///
+/// Takes the Ed25519 secret key (64 bytes), unix timestamp, method, path, and optional body.
+/// Returns a version-blinded signature.
+LIBSESSION_EXPORT bool session_blind_version_sign_request(
+        const unsigned char* ed25519_seckey, /* 64 bytes */
+        size_t timestamp,
```

Comment:
A timestamp value should ideally be a 64-bit type; probably an `int64_t`.  (`size_t` is only 32-bits on 32-bit platforms like ARMv7 and x86).

Reply:
Applies to `session_blind_version_sign` as well, fixed both.